src/pages/partners/affiliate-ib/_deriv-affiliate-programme.js

```diff
@@ -116,30 +99,47 @@ const CPAContent = styled.div`
 `
 
 const StyledCard = styled(Card)`
+    min-width: calc((100% - 4.8rem) / 3);
+    width: calc((100% - 4.8rem) / 3);
     border-radius: 8px;
 
     :nth-child(2) {
-        margin: 1.6rem 2.4rem 0;
+        margin: 1.6rem 2.8rem 0 2rem;
 
         @media ${device.tabletL} {
-            margin: 24px 0;
+            margin: 20px 0 11px;
         }
     }
 
+    @media ${device.laptopM} {
+        min-width: 38.4rem;
+        width: 38.4rem;
+    }
+
     @media ${device.tabletL} {
-        min-width: 328px;
-        padding: 16px;
+        min-width: unset;
+        padding: 24px 16px;
         height: ${(props) => (props.tabletHeight ? props.tabletHeight : '')};
```

Comment:
Could we make this `tablet_height` to be consistent with `mt_mobile`?

Reply:
Amended in the latest commit